ggml/src/ggml.c

```diff
@@ -3693,6 +3758,10 @@ struct ggml_context * ggml_init(struct ggml_init_params params) {
             GGML_PRINT_DEBUG("%s: g_state initialized in %f ms\n", __func__, (t_end - t_start)/1000.0f);
         }
 
+#if defined(__aarch64__)
+        ggml_init_aarch64_features();
+#endif
+
```

Comment:
This looks incorrect because ARM NEON presence is now associated with `__aarch64__`, but this is not always the case AFAIK. For example, here we have support for `__ARM_NEON && !__aarch64__`, such as Raspberry Pi:

https://github.com/ggerganov/llama.cpp/blob/64c6af3195c3cd4aa3328a1282d29cd2635c34c9/ggml/src/ggml-cpu-impl.h#L132-L167

Reply:
I've addressed this comment in the update. Non aarch64 Linux builds fallback to using compile time flags for detecting whether or not Neon is enabled.